### PR TITLE
[CDAP-16271] Fix dropdown in control center

### PR DESCRIPTION
**Pull request by yukiej**
Before the library upgrade, we had reactstrap version 6.2.0. Issue was caused by [breaking changes in v8.0.0](https://github.com/reactstrap/reactstrap/releases) to use the new Context API. As a result, CustomDropdownMenu component wasn't getting context from Dropdown anymore.

Upgraded reactstrap version from 8.2.0 to 8.4.0 to give CustomDropdownMenu access to DropdownContext. 

Build: https://builds.cask.co/browse/CDAP-UDUT506

### CODE REVIEW

File: cdap-ui/cypress/integration/controlcenter.spec.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    // Deploy a pipeline to have a pipeline and some datasets to see
+    Helpers.deployAndTestPipeline('fll_airport_pipeline2.json', testPipeline, done);
+  });
+  after(() => {
+    // Delete the pipeline to clean up
+    cy.cleanup_pipelines(headers, testPipeline);
+  });
+  it('allows users to open filter menu and filter out pipelines', () => {
+    cy.visit('/cdap/ns/default/control');
+    cy.wait(2000);
```
Comment (elfenheart) [inline cdap-ui/cypress/integration/controlcenter.spec.ts:46]: we should stop doing explicit waits. `cy.get` have an implicit wait until the element shows up. There is a timeout that you can override on the get method.

Comment (yukiej) [reply cdap-ui/cypress/integration/controlcenter.spec.ts:46]: Agreed. That said, I tried adding an implicit wait to the cy.get here, and for some reason this is not working. I was able to remove the second explicit wait though, as well as a wait in the deployment of the pipeline. 

I added a to-do to comment to further investigate this and other oddities in this test later.

File: cdap-ui/cypress/helpers.ts [hunk 2]
```diff
@@ -113,7 +113,8 @@ function deployAndTestPipeline(filename, pipelineName, done) {
     .type(pipelineName)
     .type('{enter}');
   cy.get('[data-testid=deploy-pipeline]').click();
-  cy.wait(10000);
+  // cy.wait(10000);
```
Comment (elfenheart) [inline cdap-ui/cypress/helpers.ts]: nit: remove

**Review by elfenheart — APPROVED** [review]
Minor nit, otherwise 👍 LGTM

**Comment by yukiej** [comment]
Adding for posterity and future improvements: When I took out the first cy.wait and added a timeout to the following cy.get instead, cypress failed to find the element initially. It would try again later and eventually succeed, but for some reason the whole test would time out/fail even if all the assertions in the test passed.

**Comment by yukiej** [comment]
e2e tests: https://builds.cask.co/browse/IT-UPD2115

**Comment by yukiej** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-UDUT506-6

**Pull request merged**